Core/GDCore/Events/Parsers/ExpressionParser2Node.cpp

```diff
@@ -6,5 +6,5 @@
 #include "ExpressionParser2Node.h"
 
 namespace gd {
-gd::String ExpressionParserDiagnostic::noMessage = "";
+
 }
```

Comment:
This file is now empty, but I guess it may be useful one day.
Should I remove it?

Reply:
You can keep it :)